### PR TITLE
WAV-1410

### PR DESCRIPTION
**Pull request by MrGraversen**
**Jira:** https://wavely.atlassian.net/browse/WAV-1410

Originally a task to enable some customer desires, turned into a task about enabling customers to read how to do what they want. And a general catching-up of the documentation.

### CODE REVIEW

File: docs/general-information/data-types.md [hunk 1]
```diff
@@ -61,9 +60,24 @@ Enumeration, may have the following values:
...
+Enumeration, may have the following values:
+
+* KBIT_32
+* KBIT_64
+* KBIT_128
+* KBIT_256
+* KBIT_512
+* MBIT_1
+* MBIT_2
+* MBIT_5
```
Comment (grandegiraffo) [inline docs/general-information/data-types.md]: No full or fast speed?

**Review by grandegiraffo — CHANGES_REQUESTED** [review]

File: docs/general-information/data-types.md (hunk 1 above)
Comment (MrGraversen) [reply docs/general-information/data-types.md]: I relied on the API documentation as truth source. It is defined like so:

```java
    @ApiObjectField(
            required = false,
            allowedvalues = {"KBIT_32", "KBIT_64", "KBIT_128", "KBIT_256", "KBIT_512", "MBIT_1", "MBIT_2", "MBIT_5"}
    )
    private NetworkSpeeds networkSpeed;
```

Comment (grandegiraffo) [reply docs/general-information/data-types.md]: 🤦 I momentarily forgot how we unthrottle. Carry on.

**Review by grandegiraffo — APPROVED** [review]

File: docs/general-information/wavely-api-docs.md [hunk 2]
```diff
@@ -35,6 +33,12 @@ Then your basic auth header should look like this:
 Authorization: Basic N2RlZmViM2ItZjAzYS00Y2Q4LWEzYzItZmVhNDVhMjVhYmU0OnRveXFkQ0FMRmFpMTl1V29mSURseGI2aHA3WmhPbGFmOXN4NWhURUsxR0lPSktVUjN5NHdBa1BUcnp2U3lISno=
 ```
 
+## Adding an API key
+
+Go to your account and go to your _Advanced_ settings, to create an API key. See below:
+
+![add-api-key](../img/webhooks/add-api-key.png)
```
Comment (Dickow) [inline docs/general-information/wavely-api-docs.md:40]: Wrong folder in terms of context, but that may not matter too much.

**Review by Dickow — APPROVED** [review]

**Pull request merged**